main.go

```diff
@@ -72,7 +75,7 @@ func (c *influxDBCollector) serveUdp() {
 	for {
 		n, _, err := c.conn.ReadFromUDP(buf)
 		if err != nil {
-			log.Warnf("Failed to read UDP message: %s", err)
+			level.Warn(c.logger).Log("msg", "Failed to read UDP message", "warn", err)
```

Comment:
is this the way it's supposed to be done in go-kit? We are already logging at warning level here, so I don't think we need a "warn" key to tell us that again? I'd rather use `error` as in the other log lines:
```suggestion
			level.Warn(c.logger).Log("msg", "Failed to read UDP message", "error", err)
```

Reply:
`err` is the usual key.

Reply:
ah! updated the suggestion